### ISSUE TITLE
Google Sheets Importer - compiler error

### ISSUE THREAD

**Issue by 1TommyBoy** [FrozenTundraTrader/sierrachart#9]
Hi @FrozenTundraTrader.  When I run the compiler on the "Google_Sheets_Importer.cpp", I get an error at line #312 in the result output:
(312): "error C2440: 'initializing': cannot convert from 'const char *' to 'char *'

Any thought as to what the issue may be on my end?

**Comment by FrozenTundraTrader** [comment]
@1TommyBoy likely compiler settings you're using are stricter than Sierra Charts remote compiler. You using visual studio?

**Comment by 1TommyBoy** [comment]
Yes, I'm using VS 2022.  Sierra Charts remote compiler doesn't work on my system all of the time.  I'll try to resolve that and compile it through the remote option.  Thanks for the help.

**Comment by FrozenTundraTrader** [comment]
@1TommyBoy other option is to modify the source code to comply with your compiler settings.
